commerce-checkout-web/src/main/resources/META-INF/resources/checkout_step/address.jsp

```diff
@@ -188,6 +188,8 @@ long commerceRegionId = BeanParamUtil.getLong(currentCommerceAddress, request, "
 				}
 				else {
 					<portlet:namespace />updateAddressFields(commerceAddress.get('selectedIndex'));
+
+					Liferay.Form.get("<portlet:namespace />fm").formValidator.validate();
```

Comment:
Actually can you change these to single quotes? Looks like every other JS string in this file is single-quoted, so it'd be great to keep it consistent.